lightningd.py

```diff
@@ -55,7 +55,7 @@ class LightningNode(object):
 
     displayName = 'lightning'
 
-    def __init__(self, lightning_dir, lightning_port, btc, executor=None, node_id=0):
+    def __init__(self, lightning_dir, lightning_port, btc, electrumx, executor=None, node_id=0):
```

Comment:
Why add the dependency on `electrumx` if `lightningd` can run perfectly fine without it?